qemu/tests/driver_in_use.py

```diff
@@ -96,7 +96,7 @@ def run_subtest(sub_type):
     sub_type = params["sub_test"]
     bg_stress_test = params["run_bgstress"]
     wait_time = float(params.get("wait_bg_time", 60))
-    suppress_exception = bool(params.get("suppress_exception", "no"))
+    suppress_exception = bool(params.get("suppress_exception", "no") == "yes")
```

Comment:
really no need to do it.  suppress_exception = params.get("suppress_exception", "no") == "yes" is enough